<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Agency FB&quot;;
color: rgb(255, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Adobe Devanagari&quot;;</string>
   </property>
   <property name="text">
    <string>剩余时间：</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>541</width>
     <height>261</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_4">
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>200</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Adobe Devanagari&quot;;
font: 16pt &quot;Adobe Devanagari&quot;;</string>
     </property>
     <property name="text">
      <string>开始</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Adobe Devanagari&quot;;
font: 14pt &quot;Adobe Devanagari&quot;;</string>
     </property>
     <property name="text">
      <string>注意：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>431</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Adobe Devanagari&quot;;</string>
     </property>
     <property name="text">
      <string>如果你设置了小数模式，答案的精度需要与你设置的精度相同；</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>501</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Adobe Devanagari&quot;;
font: 12pt &quot;Adobe Devanagari&quot;;</string>
     </property>
     <property name="text">
      <string>每道题你将有20秒的时间进行回答，计时结束系统将提取当前答案；</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>160</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Adobe Devanagari&quot;;</string>
     </property>
     <property name="text">
      <string>点击确定，页面将跳入下一题或结束；</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>135</y>
       <width>501</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Adobe Devanagari&quot;;</string>
     </property>
     <property name="text">
      <string>如果你设置了分数模式，答案大于1时请用带分数表示（形如1'2/3）；</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Adobe Devanagari&quot;;</string>
     </property>
     <property name="text">
      <string>题目</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>521</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Adobe Devanagari&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="answer1">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>160</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Adobe Devanagari&quot;;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>210</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Adobe Devanagari&quot;;</string>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
